pkg/proto/eth_transaction.go

```diff
@@ -215,7 +239,7 @@ func (tx *EthereumTransaction) Verify() (*EthereumPublicKey, error) {
 	if senderPK := tx.threadSafeGetSenderPK(); senderPK != nil {
 		return senderPK, nil
 	}
-	signer := MakeEthereumSigner(tx.ChainId())
+	signer := MakeEthereumSigner(tx.inner.chainID())
```

Comment:
Use `tx.inner.chainID()` in all places and remove `tx.ChainID()` method. Or use `tx.ChainId()` and don't use `tx.inner.chainID()` anywhere.